Lib/fontTools/ttLib/removeOverlaps.py

```diff
@@ -76,6 +81,37 @@ def ttfGlyphFromSkPath(path: pathops.Path) -> _g_l_y_f.Glyph:
     return glyph
 
 
+def _simplify(path: pathops.Path, debugGlyphName: str) -> pathops.Path:
+    # skia-pathops has a bug where it sometimes fails to simplify paths when there
+    # are float coordinates and control points are very close to one another.
+    # Rounding coordinates to integers works around the bug.
+    # Since we are going to round glyf coordinates later on anyway, here it is
+    # ok(-ish) to also round before simplify. Better than failing the whole process
+    # for the entire font.
+    # https://bugs.chromium.org/p/skia/issues/detail?id=11958
+    # https://github.com/google/fonts/issues/3365
+    try:
+        path2 = pathops.simplify(path, clockwise=path.clockwise)
+    except pathops.PathOpsError:
+        path2 = pathops.Path()
+        for verb, points in path:
+            path2.add(verb, *((otRound(p[0]), otRound(p[1])) for p in points))
+        try:
+            path2.simplify(clockwise=path.clockwise)
+        except pathops.PathOpsError as e:
+            path.dump()
+            raise RemoveOverlapsError(
+                f"Failed to remove overlaps from glyph {debugGlyphName!r}"
+            ) from e
+        else:
+            log.debug(
+                "skia-pathops failed to simplify '%s' with float coordinates, "
+                "but succeded using rounded integer coordinates",
+                debugGlyphName,
+            )
+    return path2
```

Comment:
Optional, I found this a little hard to read. Spitballing, what about:

```python
try:
  # Normally simplify works fine
  return pathops.simplify(path, clockwise=path.clockwise)
except pathops.PathOpsError:
  pass

# Sometimes simplify fails on a path but works when rounded
path = _round_path(path)

try:
  # Original code did pathops.simplify one time, path.simplify the second, not sure if that was significant
  path = pathops.simplify(path, clockwise=path.clockwise)
  log.debug("skia-pathops failed to simplify etc etc")
  return path
except pathops.PathOpsError:
  # dump and re-raise

assert False, "Unreachable"
```

Reply:
Done